content/operate/kubernetes/architecture/_index.md

```diff
@@ -5,44 +5,152 @@ categories:
+
+Kubernetes is a dynamic environment, with nodes and pods changing as needed. The services rigger monitors the cluster for these changes and updates the database services to ensure reliable communication with the databases.
+
+## Active-Active databases
+
+On Kubernetes, Redis Enterprise [Active-Active]({{< relref "/operate/rs/databases/active-active/" >}}) databases provide read and write access to the same dataset from different Kubernetes clusters. Creating an Active-Active database requires routing [network access]({{< relref "/operate/kubernetes/networking/" >}}) between two Redis Enterprise clusters residing in different Kubernetes clusters. Without the proper access configured for each cluster, syncing between the databases instances will fail.The admission controller is also required for Active-Active databases on Kubernetes, to validate changes to the custom resources.
+
+For more details and installation information, see [Active-Active databases]({{<relref "/operate/kubernetes/active-active/">}}). For more general information about Active-Active, see the [Redis Enterprise Software docs]({{< relref "/operate/rs/databases/active-active/" >}}).
+
+## RedisEnterpriseRemoteCluster RERC
+
+The [RedisEnterpriseRemoteCluster (RERC)]({{<relref "/operate/kubernetes/reference/redis_enterprise_remote_cluster_api">}}) contains details allowing the REC to link to the RedisEnterpriseActiveActiveDatabase (REAADB). The RERC resource is listed in the [REAADB](#redisenterpriseactiveactivedatabase-reaadb) resource to become a participating cluster for the Active-Active database.
+
+See the [RERC API reference]({{<relref "/operate/kubernetes/reference/redis_enterprise_remote_cluster_api">}}) for a full list of fields and settings.
+
+## RedisEnterpriseActiveActiveDatabase REAADB
+
+The RedisEnterpriseActiveActiveDatabase (REAADB) resource creates and manages a database that spans more than one Kubernetes cluster. An REAADB requires [external routing](#networking), at least two [RECs](#redisenterprisecluster-rec), and at least two [RERCs](#redisenterpriseremotecluster-rerc).
+
+See the [REAADB API reference]({{<relref "/operate/kubernetes/reference/redis_enterprise_active_active_database_api">}}) for a full list of fields and settings.
+
+## Metrics
+
+To collect  metrics data from your databases and Redis Enterprise cluster (REC), you can [connect your Prometheus]({{<relref "c/operate/kubernetes/re-clusters/connect-prometheus-operator">}}) server to an endpoint exposed on your REC. Redis Enterprise for Kubernetes creates a dedicated service to expose the `prometheus` port (8070) for data collection. A custom resource called `ServiceMonitor` allows the [Prometheus operator](https://github.com/prometheus-operator/prometheus-operator/tree/main/Documentation) to connect to this port and collect data from Redis Enterprise.
```

Comment:
I think the end of this page should have a section for Deployment options that links to the child page, it could be under the heading Next Steps even, but I think the page could be missed otherwise.